pkg/agent/flowexporter/connections/conntrack.go

```diff
@@ -59,7 +57,7 @@ func filterAntreaConns(conns []*flowexporter.Connection, nodeConfig *config.Node
 		// Therefore, we ignore the connection with ClusterIP and keep the connection with Endpoint Pod IP.
 		// Conntrack flows will be different for Pod-to-Service flows w/ Antrea-proxy. This implementation will be simpler, when the
 		// Antrea proxy is supported.
```

Comment:
do we need to revise this comment? I feel like we will always need this check unless AntreaProxy becomes the *only* option.